web/src/compose_recipient.js

```diff
@@ -0,0 +1,107 @@
+import $ from "jquery";
+
+import render_compose_stream_dropdown_privacy_icons from "../templates/compose_stream_dropdown_privacy_icons.hbs";
+
+import * as compose from "./compose";
```

Comment:
We can skip this import if we move a couple functions from `compose.js` here and thus actually avoid introducing the main import cycle; done this in my branch.

There will still be one with `compose_state` and this directly importing each other.